src/plumtree_broadcast.erl

```diff
@@ -117,6 +125,10 @@
 %% to generate membership updates as the ring changes.
 -spec start_link() -> {ok, pid()} | ignore | {error, term()}.
 start_link() ->
+    LazyTickPeriod = application:get_env(plumtree, lazy_tick_period,
```

Comment:
Can you use `plumtree_config` to get / set these values as the rest of the configuration is doing?